client/src/components/Common/AppBar/index.js

```diff
@@ -61,7 +61,24 @@ const Username = styled.h3`
   margin: 0;
 `;
 
-const AppBar = ({ username = "Username", color = theme1 }) => {
+const CenterHeading = styled.p`
```

Comment:
This AppBar is used in multiple places, and we don't always need the extra details. What you can do is create a similar component as the existing Header from the EventEdit page. You'll want the styling to match that component, but you will not need the editable fields or the row of buttons.